src/rpc/daemon_handler.cpp

```diff
@@ -263,20 +263,42 @@ namespace rpc
 
   void DaemonHandler::handle(const SendRawTx::Request& req, SendRawTx::Response& res)
   {
-    auto tx_blob = cryptonote::tx_to_blob(req.tx);
+    handleTxBlob(cryptonote::tx_to_blob(req.tx), req.relay, res);
+  }
+
+  void DaemonHandler::handle(const SendRawTxHex::Request& req, SendRawTxHex::Response& res)
+  {
+    std::string tx_blob;
+    if(!epee::string_tools::parse_hexstr_to_binbuff(req.tx_as_hex, tx_blob))
+    {
+      LOG_PRINT_L0("[SendRawTxHex]: Failed to parse tx from hexbuff: " << req.tx_as_hex);
```

Comment:
New code should really use severity based calls, like MERROR/MWARNING etc.

Reply:
Fixed, should I change all the LOG_PRINT_L0 calls elsewhere in this file while we're at it?

Reply:
I wouldn't tbh. I suppose it makes things a bit messy when you want to enable those though, so if you did I wold not complain either. But if consistency was prefered, I would leave that one to LOG_PRINT_L0 on balance. Subjectivity all around... :)

Reply:
`find monero/src -type f -print0 | xargs -0 perl -pi -le 's/LOG_ERROR/MERROR/g; s/LOG_PRINT_L(\d)/"M" . ("WARNING","INFO","DEBUG","TRACE","TRACE")[$1]/ge;'` is also an option :)

Reply:
No, thanks :)